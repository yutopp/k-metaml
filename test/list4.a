tl [1, 2, 3]