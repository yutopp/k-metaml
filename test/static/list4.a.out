Search results:

Solution 1:
V:Type -->
int list
N:Int -->
0
